gTea/app/src/main/java/com/nechaieva/gtea/utils/BaseOrderProcessor.java

```diff
@@ -23,7 +23,14 @@ public void loadMenu(String[] menu) {
 
     @Override
     public Optional<String> findInMenu(String query) {
+        query = normalize(query);
```

Comment:
As mentioned in the other commit, consider being consistent with the name/description of the commit